### PR TITLE
add production env config plus refactors public display and slack commands

### PR DESCRIPTION
**Pull request by bermannoah**
-- environment reconfigured for use on digital ocean (ultimately just streamlining figaro)
-- /coffee_brewing reconfigured for public use - Turing specific language is gone now
-- CSS cleaned up on index view page
-- /last_brewed results are now scoped by team
-- removed Turing specific content/language from the how to brew page
-- updates README with new features, a how-to, and technical details (updated images coming soon™)

### CODE REVIEW

File: app/models/brew.rb [hunk 1]
```diff
@@ -58,28 +58,28 @@ def self.how_to_brew(params)
...
   end
   
   def self.we_are_out_of_coffee(params)
     {
-      "text": "Hey #{params["user_name"]} thanks for letting us know!",
+      "text": "Hey #{params["user_name"]} thanks for being so conscientious!",
       "attachments": [
         {
-          "text":"For now, please let your SAB reps know so they can update the trello board."
+          "text":"...but this feature is still in development."
```
Comment (bermannoah) [inline app/models/brew.rb]: would it be better to just remove this functionality for now?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <%= render 'application/favicon' %>
-    <title>418 Bot</title>
+    <title>Coffee-Bot</title>
```
Comment (bermannoah) [inline app/views/layouts/application.html.erb]: okay should it be coffee-bot or just coffee bot

File: config/environments/production.rb [hunk 3]
```diff
@@ -42,7 +42,10 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+
+  config.logger       = Logger.new(STDOUT)
```
Comment (bermannoah) [inline config/environments/production.rb]: This is what makes the Unicorn logs *slightly* more verbose

File: app/services/slack_service.rb [hunk 4]
```diff
@@ -19,24 +19,27 @@ def self.redirect_uri
...
+	faraday.request  :url_encoded
         faraday.adapter  Faraday.default_adapter
-        faraday.basic_auth(ENV['API_KEY'], ENV['API_SECRET'])
+        faraday.basic_auth(ENV['SLACK_API_KEY'], ENV['SLACK_API_SECRET'])
       end
     end
   
     def self.fetch_info_via_code(code)
-      response = conn.get do |req|
+	print(ENV['API_KEY'])
```
Comment (NickyBobby) [inline app/services/slack_service.rb]: why is this here?

File: app/views/coffee/index.html.erb [hunk 5]
```diff
@@ -1,23 +1,24 @@
 <div class="text-center">
 
-<h2>Welcome to ☕️ 418-Bot!</h2>
+<h2>Welcome to ☕️ Coffee-Bot!</h2>
+
 
   <% if current_user.nil? %>
-    <a href="https://slack.com/oauth/authorize?scope=identity.basic,identity.team&client_id=2329094327.104681505318"><img alt="Sign in with Slack" height="40" width="172" src="https://platform.slack-edge.com/img/sign_in_with_slack.png" srcset="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x" /></a>
+    <a href="https://slack.com/oauth/authorize?scope=identity.basic,identity.team&client_id=107333465906.108008566694"><img alt="Sign in with Slack" height="40" width="172" src="https://platform.slack-edge.com/img/sign_in_with_slack.png" srcset="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x" /></a>
```
Comment (NickyBobby) [inline app/views/coffee/index.html.erb]: is the client_id here ok to be here?

File: app/services/slack_service.rb (hunk 4 above)
Comment (bermannoah) [inline app/services/slack_service.rb]: whoops. fixed. more anonymous now.

File: app/views/coffee/index.html.erb (hunk 5 above)
Comment (bermannoah) [inline app/views/coffee/index.html.erb]: yup -- it's Slack's code + it's a public key. not very anonymous :(

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (NickyBobby) [inline app/views/layouts/application.html.erb]: I like Coffee-Bot!

File: app/models/brew.rb [hunk 6]
```diff
@@ -36,9 +36,10 @@ def brewed_coffee_response(params)
     }
   end
   
-  def self.get_last_brewed(limit)
+  def self.get_last_brewed(limit, params)
     list = ''
-    Brew.order(created_at: :desc).limit(limit).each do |brew|
+    team = Team.find_by(team_slack_id: params["team_id"])
+	Brew.where(team_id: team.id).order(created_at: :desc).limit(limit).each do |brew|
```
Comment (NickyBobby) [inline app/models/brew.rb]: could you do team.brews instead of Brew.where(team_id: team.id)?

Comment (NickyBobby) [inline app/models/brew.rb]: But it SHOULD be anonymous...

Comment (bermannoah) [inline app/models/brew.rb]: magic!

File: app/models/slack_login_user.rb [hunk 7]
```diff
@@ -15,4 +15,4 @@ def self.create_from_slack(user_info)
       user.save ? user : false
   end    
   
-end
```
Comment (NickyBobby) [inline app/models/slack_login_user.rb]: what was this?

File: app/models/brew.rb (hunk 6 above)
Comment (bermannoah) [inline app/models/brew.rb]: update: did the thing

**Comment by NickyBobby** [comment]
I wish it was just a little bit more anonymous but this is GTM 👍
